generator/src/test/java/com/scottlogic/deg/generator/fieldspecs/relations/FieldSpecRelationsTest.java

```diff
@@ -69,14 +75,22 @@ public void afterOrAt_range_returnsFromMin(){
     }
 
     @Test
-    public void after_range_returnsFromMin(){
-        FieldSpec fieldSpec = forYears(2018, 2021);
+    public void after_range_returnsFromMin() {
```

Comment:
The functionality has changed a bit here because the granularity is no longer respected.  When used in FieldSpecGroupValueGenerator the correct granularity is applied by merging two field specs.